certbot-apache/setup.py

```diff
@@ -11,7 +11,7 @@
     'acme=={0}'.format(version),
     'certbot=={0}'.format(version),
     'mock',
-    'python-augeas<=0.5.0',
+    'python-augeas',
```

Comment:
This was discussed with a lot of detail in #4433. Basically our plan is not to specify the maximum version of our dependencies on PyPI as when new versions of our dependencies are released and packaged in OSes, Certbot package maintainers have to do extra work to remove these requirements.

Instead, the plan is to continue pinning specific versions of dependencies in `certbot-auto` and add this to our developer set up so temporary problems with new versions of upstream dependencies don't slow down Certbot contributors. We have other tests that test with the newest versions of all our dependencies so core developers will still be notified when changes from upstream break things.